specification/components/schemas/contractCertificateData.v1.json

```diff
@@ -13,8 +13,8 @@
             "$ref": "../parameters/xsdMsgDefNamespaceParam.v1.json"
         },
         "contractCertificate": {
-            "type": "string",
-            "description": "Contract Certificate in base64 encoded DER form",
+            "type": "MIICQDCCAeWgAwIBAgICBNMwCgYIKoZIzj0EAwIwRDELMAkGA1UEBhMCREUxDTALB",
```

Comment:
type needs to stay "string", we can add "format : byte"  (https://swagger.io/docs/specification/data-models/data-types/#string)

Reply:
correct, mistake on my side